src/entity-type-def-manager.ts

```diff
@@ -349,29 +367,17 @@ export class EntityTypeDefManager {
             ) as Nexus.core.AllInputTypes
             if (columnTypeName === 'String') {
               singleOperandOperations.forEach(singleOperandOperation => {
-                t.field(`${column.propertyName}_${singleOperandOperation}`, {
-                  type: columnTypeName!,
+                t.field(column.propertyName, {
+                  type: whereStringInputType!,
```

Comment:
I didn't know we could pass the direct object to the type. To mantain a pattern for now. What do you think of including the inputObject at the [src/plugin.ts:42](https://github.com/vinpac/nexus-typeorm-plugin/blob/e09d2e58bfb361db0c7d913395815e7f6814dd18/src/plugin.ts#L42)?

```
  return [
    buildEntityFieldOutputMethod(manager),
    buildCRUDOutputMethod(manager),
    buildEntityOutputProperty(manager),
    buildEntityFieldsOutputMethod(manager),
    buildCRUDOutputProperty(manager),
    GraphQLDateTime,
    StringFilterInput,
    IntFilterInput,
  ]
```

```suggestion
                  type: 'StringFilter'
```

Reply:
Ok, no problem, it's your lib. But generally speaking I prefer using the object directly as It boast better type support and is less error prone.

Reply:
Didn't get your last comment about `plugin.ts` ? Do you want to initiate the input types from this file ?

Reply:
Yeah. Os this part we return an array defining the field creators and global types (e.g. `GraphQLDateTime`). Let's put StringFilter and IntFilter there for now so we don't have multiple implementations on how we are managing types. If later we find it better using the object directly, we migrate it.

I'm looking for reaching a stable version by the end of February.

Reply:
Not sure how to fix tests tho. I have to regenarate nexus types in the test forlder, and snapshot as well

Reply:
nvm got it

Reply:
Is your test suit supposed to pass as is ?

Reply:
No, let take one test as example.

```
it('handles simple operations', async () => {
    const result = await query(`
      query {
        users(where: {
          age_gt: 35,
          age_lt: 50
        }) {
          id
          name
          age
        }
      }`)

    expect(result).toMatchObject({
      users: [
        {
          age: 40,
          id: expect.any(String),
          name: 'baz',
        },
      ],
    })
  })
```

should change to
```
it('handles simple operations', async () => {
    const result = await query(`
      query {
        users(where: {
          age: { gt: 35, lt: 50 },
        }) {
          id
          name
          age
        }
      }`)

    expect(result).toMatchObject({
      users: [
        {
          age: 40,
          id: expect.any(String),
          name: 'baz',
        },
      ],
    })
  })
```

Reply:
No, I know :wink: ! I was refering to without my changes. Got almost all test running now, have a few things to work out yet tomorrow. Implementation wise, should I declare my input types in the `plugin.ts` file or set it up in another file like `genericTypes.ts` ? Because I also added an "id" input type (only `equals` and `in`) and more will come with adaptation of the sorting, and future relay-style pagination.

Reply:
Wow! Nice work. For now, let's declare it on `src/args/arg-where`.